vm_helper_files/Makefile

```diff
@@ -71,7 +71,7 @@ online-prep: setup
 clean:
 	scripts/cleanup.sh
 
-install-local:
+install-local: online-prep
```

Comment:
Maybe we should rename `online-prep` since this is a confusing addition for testing local offline.